src/app/tests/TestReadInteraction.cpp

```diff
@@ -54,14 +56,14 @@ uint8_t gDebugEventBuffer[128];
 uint8_t gInfoEventBuffer[128];
 uint8_t gCritEventBuffer[128];
 chip::app::CircularEventBuffer gCircularEventBuffer[3];
-chip::ClusterId kTestClusterId          = 6;
-chip::ClusterId kTestEventClusterId     = chip::Test::MockClusterId(1);
-chip::ClusterId kInvalidTestClusterId   = 7;
-chip::EndpointId kTestEndpointId        = 1;
-chip::EndpointId kTestEventEndpointId   = chip::Test::kMockEndpoint1;
-chip::EventId kTestEventIdDebug         = chip::Test::MockEventId(1);
-chip::EventId kTestEventIdCritical      = chip::Test::MockEventId(2);
-uint8_t kTestFieldValue1                = 1;
+chip::ClusterId kTestClusterId        = 6;
+chip::ClusterId kTestEventClusterId   = chip::Test::MockClusterId(1);
+chip::ClusterId kInvalidTestClusterId = 7;
+chip::EndpointId kTestEndpointId      = 1;
+chip::EndpointId kTestEventEndpointId = chip::Test::kMockEndpoint1;
+chip::EventId kTestEventIdDebug       = chip::Test::MockEventId(1);
+chip::EventId kTestEventIdCritical    = chip::Test::MockEventId(2);
+// uint8_t kTestFieldValue1                = 1;
```

Comment:
shoudl we delete things instead of commenting them out? Where was this used before?

Reply:
i will delete it, this was here before, and now I moved it to `test-interaction-model-api` where it is used by the function `ReadSingleClusterData`